src/ProjectNeon/Assets/Features/Battle/BarkskinStats.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BarkskinStats : BattleStats
+{
+    private float quantity = 10F;
+
+    public new float Armor()
```

Comment:
Okay, this concept can work.

`new` doesn't work though. It only applies when the runtime is expecting a `Barkskin` and is ignored when he is used by his interface or abstract class.

You can use `virtual` in the base class and `override` in the child if we want to go this route.

I have some concern, since this violate encapsulation, but since this is data, it's probably fine.